test/e2e/page-objects/pages/asset-list.ts

```diff
@@ -0,0 +1,86 @@
+import { Driver } from '../../webdriver/driver';
+
+class AssetListPage {
+  private readonly driver: Driver;
+
+  // Selectors
+
+  private readonly networksToggle = '[data-testid="network-filter"]';
+
+  private readonly allNetworksOption =
+    '[data-testid="network-filter-all__button"]';
+
+  private readonly currentNetworkOption =
+    '[data-testid="current-network__button"]';
+
+  private readonly allNetworksTotal =
+    '[data-testid="network-filter-all__total"]';
+
+  private readonly currentNetworksTotal = `${this.currentNetworkOption} [data-testid="account-value-and-suffix"]`;
+
+  constructor(driver: Driver) {
+    this.driver = driver;
+  }
+
+  async openNetworksFilter(): Promise<void> {
+    console.log(`Opening the network filter`);
+    await this.driver.clickElement(this.networksToggle);
+  }
+
+  async getNetworksFilterLabel(): Promise<string> {
+    console.log(`Retrieving the network filter label`);
+    const toggle = await this.driver.findElement(this.networksToggle);
+    const text = await toggle.getText();
+    return text;
+  }
+
+  async clickCurrentNetworkOption(): Promise<void> {
+    console.log(`Clicking on the current network option`);
+    await this.driver.clickElement(this.currentNetworkOption);
+  }
+
+  async getAllNetworksOptionTotal(): Promise<string> {
+    console.log(`Retrieving the "All networks" option fiat value`);
+    const allNetworksValueElement = await this.driver.findElement(
+      this.allNetworksTotal,
+    );
+    const value = await allNetworksValueElement.getText();
+    return value;
+  }
+
+  async getCurrentNetworksOptionTotal(): Promise<string> {
+    console.log(`Retrieving the "Current network" option fiat value`);
+    const allNetworksValueElement = await this.driver.findElement(
+      this.currentNetworksTotal,
+    );
+    const value = await allNetworksValueElement.getText();
+    return value;
+  }
+
+  async selectNetworkFilterAllNetworks(): Promise<void> {
+    console.log(`Selecting "All networks" from the network filter`);
+    await this.driver.clickElement(this.allNetworksOption);
+
+    // TODO: Replace this with polling for text to say "All networks"
+    await this.driver.delay(2000);
+  }
+
+  async selectNetworkFilterCurrentNetwork(): Promise<void> {
+    console.log(`Selecting "Current network" from the network filter`);
+    await this.driver.clickElement(this.currentNetworkOption);
+
+    // TODO: Replace this with polling for text to *not* say "All networks"
```

Comment:
Surprised to still see these here, we should have been able to poll for this at the beginning of the week.